tasks/main.yml

```diff
@@ -1,9 +1,17 @@
 ---
-- include: "debian.yml"
-  when: ansible_os_family == "Debian"
+- include: "install_debian.yml"
+  when: ansible_os_family == "Debian" and strongloop_install
 
-- include: "redhat.yml"
-  when: ansible_os_family == "RedHat"
+- include: "install_redhat.yml"
+  when: ansible_os_family == "RedHat" and strongloop_install
+
+#
+# Add the support for multi versions of the same code
+#
+- include: strongloop_deploy.yml
```

Comment:
you can remove `strongloop` from the file name - you are within the role, we don't expect nothing but strongloop to be deployed.